addons/mail/models/mail_channel_partner.py

```diff
@@ -66,6 +66,25 @@ def write(self, vals):
         return super(ChannelPartner, self).write(vals)
 
     def unlink(self):
+        for channel_partner in self:
+            notification = {
+                'type': 'rtc_sessions_update',
+                'payload': {
+                    'id': channel_partner.channel_id.id,
+                    'rtcSessions': [('insert-and-unlink', {'id': channel_partner.sudo().rtc_session_ids.id})],
```

Comment:
Not applicable if the other comment is applied, but in this format it would send notifications even if the channel partner didn't have any session.